### PR TITLE
Add Summary row to Gitpod overview dashboard

### PR DESCRIPTION
**Pull request by aledbf**
## Description

By default the new row is collapsed

Gitpod-Overview-next-Grafana

## Release Notes
```release-note
NONE
```

## Werft options:

- [ ] /werft with-preview

### CODE REVIEW

**Review by jenting — APPROVED** [review]

**Review by geropl — APPROVED** [review]
@aledbf Judging from the screenshot you shared workspace types are counted separately (maybe by cluster, or nodepool)?

LGTM besides :heavy_check_mark: 

/hold

File: operations/observability/mixins/Makefile [hunk 1]
```diff
@@ -1,44 +1,48 @@
...
-.PHONY: promtool-lint
-promtool-lint: prometheus_alerts.yaml
+promtool-lint: prometheus_alerts.yaml # Check generated prometheus alerts yaml file
 	promtool check rules prometheus_alerts.yaml
 
-.PHONY: vendor
 vendor:
 	jb install
+	# Custom version of https://github.com/grafana/grafonnet-lib/pull/274
+	cp components/bar_gauge_panel.libsonnet vendor/github.com/grafana/grafonnet-lib/grafonnet/bar_gauge_panel.libsonnet
```
Comment (ArthurSens) [inline operations/observability/mixins/Makefile:36]: This works when developing locally, but won't work when importing those changes to our observability repository

We have another vendor file over there and it we can't move directories around when rendering dashboards with ArgoCD 😕

File: operations/observability/mixins/cross-teams/dashboards/gitpod-overview.libsonnet [hunk 2]
```diff
@@ -9,6 +9,7 @@ local row = grafana.row;
 local prometheus = grafana.prometheus;
 local template = grafana.template;
 local graphPanel = grafana.graphPanel;
+local barGaugePanel = grafana.barGaugePanel;
```
Comment (ArthurSens) [inline operations/observability/mixins/cross-teams/dashboards/gitpod-overview.libsonnet:12]:
```suggestion
local barGaugePanel = import '../../components/bar_gauge_panel.libsonnet';
```

This way you don't need the workaround of moving directories inside the vendor folder 🙂

**Review by ArthurSens — CHANGES_REQUESTED** [review]
Overall it looks good!

It just needs a small change because we cannot make changes to the `vendor` folder. This folder is always created by automation with [jsonnet-bundler](https://github.com/jsonnet-bundler/jsonnet-bundler) and whatever changes we make here will be overridden when going to production.

Also, I've noticed that people are more comfortable with raw JSON. Do you wanna keep using jsonnet for this dashboard?

File: operations/observability/mixins/cross-teams/dashboards/gitpod-overview.libsonnet (hunk 2 above)
Comment (aledbf) [reply operations/observability/mixins/cross-teams/dashboards/gitpod-overview.libsonnet]: Fixed

**Review by ArthurSens — APPROVED** [review]

**Comment by aledbf** [comment]
> Do you wanna keep using jsonnet for this dashboard?

I just want to add a Summary. If someone wants to own such change across the dashboards,  go :+1:

**Pull request merged**